src/golang/lib/workflow/operator/service.go

```diff
@@ -27,14 +27,15 @@ func GetOperatorsOnIntegration(
 		return operatorRepo.GetExtractAndLoadOPsByIntegration(ctx, integrationID, DB)
 	}
 
-	// This feature is not supported for the given service.
-	if !shared.IsComputeIntegration(integrationObject.Service) {
-		return nil, nil
+	if _, ok := shared.ServiceToEngineConfigIntegrationIDField[integrationObject.Service]; ok {
+		return operatorRepo.GetByEngineIntegrationID(ctx, integrationID, DB)
 	}
 
+	// Other eligible cases
 	if integrationObject.Service == shared.Conda {
 		return operatorRepo.GetWithExecEnv(ctx, DB)
 	}
 
-	return operatorRepo.GetByEngineIntegrationID(ctx, integrationID, DB)
+	// This feature is not supported for the given service.
+	return nil, nil
```

Comment:
Currently, which service do we not support? And if so should we error instead of returning nothing?

Reply:
Notifications for example. I think it's fine to return an empty list for now, mainly that we don't have much checking in frontend / SDK to ensure we don't call any API that involves with this query. So if we want to return an error we need to proceed really carefully.